website/pages/docs/installation/_meta.json

```diff
@@ -2,6 +2,7 @@
   "cli": "CLI",
   "postcss": "PostCSS",
   "astro": "Astro",
+  "ember": "Ember",
```

Comment:
Is there some desired order here? I assume `CLI` and `PostCSS` should be at the top, the rest is semi-alphabetical?